icepyx/core/read.py

```diff
@@ -289,6 +289,9 @@ class Read:
     glob_kwargs : dict, default {}
         Additional arguments to be passed into the [glob.glob()](https://docs.python.org/3/library/glob.html#glob.glob)function
 
+    glob_kwargs : dict, default {}
+        Additional arguments to be passed into the [glob.glob()](https://docs.python.org/3/library/glob.html#glob.glob)function
+
```

Comment:
```suggestion
```

merge issue?

Reply:
it might be the diff view. We had it happen in another PR that when we changed the base it mucked up the diff (.. vs ...; I'll try changing the base to see if we can get it to reset)

Reply:
Note: remove duplicate docstring entry